src/sample/Controller.java

```diff
@@ -1,4 +1,63 @@
 package sample;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.control.ListView;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.HBox;
 
-public class Controller {
+import java.net.URL;
+import java.util.ResourceBundle;
+
+public class Controller implements Initializable {
+
+    ObservableList<Contact> items = FXCollections.observableArrayList();
+
+    @FXML
+    ListView list;
+
+    @FXML
+    TextField Name;
+
+    @FXML
+    TextField Phone;
+
+    @FXML
+    TextField Email;
+
+
+    @FXML
+    HBox text;
+
+
+    public void addItem(){
+        Contact contact = new Contact(Name.getText(), Phone.getText(), Email.getText());
```

Comment:
You need to add some code before line 35 to make sure the user can not add a contact if ANY of the fields are empty.  Then resubmit.